src/sentry/utils/arroyo.py

```diff
@@ -157,43 +158,36 @@ def close(self) -> None:
             self.__pool.close()
 
 
-class RunTaskWithMultiprocessing(ArroyoRunTaskWithMultiprocessing[TStrategyPayload, TResult]):
+def run_task_with_multiprocessing(
+    *,
+    pool: MultiprocessingPool,
+    function: Callable[[Message[TStrategyPayload]], TResult],
+    **kwargs: Any,
```

Comment:
technically these parameters are unchecked -- ideally they'd be forwarded along to the underlying callers but 🤷

Reply:
ah i think that's why i did it this way actually, with it being a subclass the user of this strategy/class/function has typed args

Reply:
BTW, don't let this convo be a blocker from merging, I don't really care about retaining this property